ocs_ci/ocs/constants.py

```diff
@@ -464,3 +464,7 @@
 REPO_MAPPING = {
     '4.2.0': OCP4_2_REPO
 }
+
+# Cluster name limits
+CLUSTER_NAME_MIN_CHARACTERS = 5
+CLUSTER_NAME_MAX_CHARACTERS = 17
```

Comment:
So cluster name should be between 6-16 chars If I understand correctly right?  Please can this be added to some of our README? For example here:  https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md . thanks

Reply:
It will only raise the ClusterNameLengthError() when the cluster name is < 5 or > 17 characters, so the range valid range would be 5-17 chars.

Reply:
Will be documented with #1143

Reply:
I'll add the relevant documentation under this PR now that it was dismissed